app/helpers/requests/application_helper.rb

```diff
@@ -62,7 +62,7 @@ def show_service_options_fill_in(requestable)
     end
 
     def hidden_service_options(requestable)
-      return if output_request_input(requestable)
+      #return if output_request_input(requestable)
```

Comment:
This method only returns tru if it output thing, so I do not think commenting it out will help...